vite.config.ts

```diff
@@ -24,6 +25,23 @@ export default defineConfig({
   },
   plugins: [
     react(),
+    VitePWA({
+      strategies: 'generateSW',
```

Comment:
It looks like the registerSW.js is not available in the offline cache.

<img width="596" alt="Screenshot 2024-11-22 at 21 48 18" src="https://github.com/user-attachments/assets/186245dd-9be7-4399-8226-daf80ba54bdd">

Reproduction:
- Open a incognito window
- Open Comify
- Go offline (in dev-tools)
- Refresh